assets/js/lib/test-utils/factories/executions.js

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import { faker } from '@faker-js/faker';
 import { Factory } from 'fishery';
-import { hostFactory, resultEnum, randomObjectFactory } from '.';
+import { hostFactory, randomObjectFactory } from '.';
 
 export const checksExecutionStatusEnum = () =>
   faker.helpers.arrayElement(['running', 'completed', 'requested']);
 
+const resultEnum = () =>
```

Comment:
Need to create its own specific `resultEnum`, as the values are different.
For example, the `resultEnum` in this case doesn't include `unknown`